adaptors/plugins.js

```diff
@@ -1,4 +1,6 @@
+"use strict";
+
 module.exports = [
-  require('lout'),
-  require('hapi-auth-bearer-token')
+  require(`lout`),
```

Comment:
is there a reason we're exporting an array here, rather than an object? This feels brittle (why would "lout" be array[0], for instance? Whereas returnval.lout being lout makes a whole bunch of sense)